Tools/MiniBrowserSwiftUI/Shared/BrowserApp.swift

```diff
@@ -28,6 +28,15 @@ import SwiftUI
 @main
 struct BrowserApp: App {
     var body: some Scene {
-        BrowserScene()
+        BrowserScene().commands {
+            CommandGroup(before: CommandGroupPlacement.newItem, addition: {
```

Comment:
Use trailing closures and static member syntax where possible.
```suggestion
            CommandGroup(before: .newItem) {
```